backend/api/models/performance.py

```diff
@@ -4,9 +4,10 @@
 class Performance(models.Model):
     compute_task = models.ForeignKey("ComputeTask", on_delete=models.deletion.DO_NOTHING, related_name="performances")
     metric = models.ForeignKey("Function", on_delete=models.deletion.DO_NOTHING, related_name="performances")
+    identifier = models.CharField(max_length=100)
```

Comment:
I think it's better to explicit this is not the id of the perf asset.
```suggestion
    task_output_identifier = models.CharField(max_length=100)
```

Reply:
Or maybe it's even better to use a FK to the [task output asset](https://github.com/Substra/substra-backend/blob/main/backend/api/models/computetask.py#L27) but you will have to ensure the "parent" asset is created prior the perf

Reply:
In this case I wonder if the compute task FK remains relevant or should be replaced by the task output FK

Reply:
You may have to also update https://github.com/Substra/substra-backend/blob/main/backend/api/events/sync.py